### PR TITLE
chore(liveness): move inline css to classes

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Moved most of the smaller components in the `shared/` directory to css classes

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx [hunk 1]
```diff
@@ -20,11 +20,13 @@ export const DefaultHeader = ({
   bodyText,
 }: DefaultHeaderProps): JSX.Element => {
   return (
-    <View flex="1">
-      <View color="font.primary" fontWeight="bold">
+    <View className={`${LivenessClassNames.StartScreenHeader}`}>
+      <View className={`${LivenessClassNames.StartScreenHeader}__heading`}>
```
Comment (reesscot) [inline packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx:24]: for consistency, shouldn't __heading and __body be their own classnames as well?

Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx:24]: do you mean like we should have a variable for them? We already have separate classes for them. This is just the pattern i saw we were already using in these files.

Comment (zchenwei) [reply packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx:24]: Yeah, why not creating their own classnames?

Comment (reesscot) [reply packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx:24]: Yeah, I mean separate variables for them so they aren't being created in the file

**Review by zchenwei — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]

**Pull request merged**